easybuild/easyconfigs/b/boost_histogram/boost_histogram-1.2.1-foss-2021a.eb

```diff
@@ -0,0 +1,28 @@
+easyblock = 'PythonBundle'
```

Comment:
If there's only a single Python package to be installed, it doesn't make much sense to use `PythonBundle`, we should use `PythonPackage` instead?